SU-HSDO/suhumsci
# FourKitchens Release: Sprint 15

# READY FOR REVIEW

## Summary
- SHS-125: Address noted opportunities for accessibility improvements
- SHS-87: Set up base reduced motion option

## Need Review By (Date)
- 12/15

## Urgency
low

## Steps to Test
Testing SHS-125:
1. Spin up HS-Colorful and HS-Traditional and verify the menu appears unchanged.
2. Use your keyboard to tab through the menu and verify it's possible to tab through each parent in line without the browser setting focus to hidden links in the dropdown.
3. Spin up the Music site and verify the Mega Menu is unchanged and the same tabbing behavior is consistent.

Testing SHS-87:
1. Go to a Colorful/Traditional page with animations. example: `components/hero-images-main-region`. Verify animations play (transitions, sliders, etc.)
2. Depending on what OS you use, turn on Reduced Motion in your Accessibility preferences. For Macs, open Accessibility -> Display. A screenshot is included below.
3. Visit the same page from above. Verify animations/transitions do not play but everything is still functional.

## Screenshots
![Screen Shot 2022-11-29 at 3 40 55 PM](https://user-images.githubusercontent.com/8405274/204643593-477576af-7f3d-44c0-a7c8-97d07ed72d8e.png)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
